docs/src/main/asciidoc/secret-management.adoc

```diff
@@ -2,7 +2,7 @@
 
 == Secret Management
 
-spring-cloud-azure-starter-keyvault-secrets adds Azure Key Vault as one of the Spring PropertySource, so secrets stored in Azure Key Vault could be easily used and conveniently accessed like other externalized configuration property, e.g. properties in files.
+Spring Cloud Azure construct link:https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#beans-property-source-abstraction[PropertySource] which holds secrets stored in Azure Key Vault secret.
```

Comment:
how about
... in Azure Key Vault secret
--->
... in Azure Key Vault Secrets
<img width="230" alt="image" src="https://user-images.githubusercontent.com/4465723/159673739-1fc7022e-ffa2-4ba3-9bba-1496e3817a5c.png">

Reply:
Updated.